src/adapters/postgres/user-adapter.ts

```
@@ -1030,7 +1030,7 @@ export class PostgresUserService implements IServicelocator {
     academicYearId: string,
     response: Response
   ) {
-    console.log('create shart here');
+    console.log('create shart here', request, userCreateDto, academicYearId);
```

ESLint

[error] Replace `'create·shart·here'` with `"create·shart·here"`

(prettier/prettier)